src/module/rendering/renderingmodule.cpp

```diff
@@ -106,10 +106,13 @@ void Vin::RenderingModule::RenderScene(RenderQueue& queue)
 			primitive.material->SetFloat("_ShadowBias", m_Ctx->mainLight.shadow.bias);
 		}
 
+        Matrix4x4<float> mvp = (projection * view * task.GetModel());
+        Matrix4x4<float> lsp = (lightprojection * lightview * task.GetModel());
+
 		primitive.material->SetMat4("vin_matrix_model", task.GetModel());
-		primitive.material->SetMat4("vin_matrix_mvp", (projection * view * task.GetModel()));
+		primitive.material->SetMat4("vin_matrix_mvp", mvp);
```

Comment:
Not needed anymore because material->SetMat/Float/Int etc doesn't take a ref now